notebooks/official/model_monitoring/model_monitoring.ipynb

```diff
@@ -202,7 +202,6 @@
       "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    USER_EMAIL = "marcacohen@gmail.com"  # @param {type:"string"}
</pre><p>Can we remove your email from the notebook? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/841/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
<p>good catch, fixed.</p>